tests/tests/shader/array_size_overrides.rs

```diff
@@ -11,6 +11,8 @@ const SHADER: &str = r#"
     @group(0) @binding(0)
     var<storage, read_write> output: array<u32>;
 
+    var<workgroup> testing: array<u32, 12>;
+
```

Comment:
Could you add a note why this is required and what it ensures is fixed, this seems... random lol (I know because 14 - 2, but still)